packages/headless/src/features/configuration/configuration-actions.ts

```diff
@@ -29,13 +29,19 @@ export interface UpdateBasicConfigurationActionCreatorPayload {
    * The Plaform URL to use (e.g., `https://platform.cloud.coveo.com`).
    */
   platformUrl?: string;
+
+  /**
+   * The Engine name (e.g., myEngine).
+   */
+  name?: string;
```

Comment:
If `name` does not need to be dynamic, then we should not allow changing it's value via an action.

Reply:
Yeah, it does not need to be stored in state/exposed through an action etc. etc.

Just a config option on creation.

You can't really change it after creation anyway, so...

Reply:
Ok I will remove name from the state